projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.ts

```diff
@@ -1,15 +1,17 @@
 import { Injectable } from '@angular/core';
 import FHIR from 'fhirclient';
-import Client from 'fhirclient/lib/Client';
-import { fhirclient } from 'fhirclient/lib/types';
 import { Observable } from 'rxjs';
 import { Bundle, FhirResource } from 'fhir/r4';
 
 export interface BloodPressure {
-  systolic?: number |null,
-  diastolic?: number |null
+  systolic?: number | null;
+  diastolic?: number | null;
 }
 
+type Client = ReturnType<typeof FHIR.client>;
+type ClientState = Client['state'];
+type Resource = Parameters<Client['create']>[0];
```

Comment:
This is the workaround: Creating type aliases from the types exported by `fhirclient` instead of importing them from `fhirclient/lib`